i18n/src/main/resources/bisq_easy.properties

```diff
@@ -66,17 +66,14 @@ bisqEasy.tradeWizard.progress.review=Review
 bisqEasy.tradeWizard.directionAndMarket.headline=Do you want to buy or sell Bitcoin with
 bisqEasy.tradeWizard.directionAndMarket.buy=Buy Bitcoin
 bisqEasy.tradeWizard.directionAndMarket.sell=Sell Bitcoin
-bisqEasy.tradeWizard.directionAndMarket.feedback.headline=How to build up reputation?
-bisqEasy.tradeWizard.directionAndMarket.feedback.subTitle1=You haven't established any reputation yet. \
-  For Bitcoin sellers, building reputation is crucial because buyers are required to send fiat currency first \
-  in the trade process, relying on the seller's integrity.\n\n\
-  This reputation-building process is better suited for experienced Bisq users, and you can find detailed information \
-  about it in the 'Reputation' section.
-bisqEasy.tradeWizard.directionAndMarket.feedback.gainReputation=Learn how to build up reputation
-bisqEasy.tradeWizard.directionAndMarket.feedback.subTitle2=While it''s possible for sellers to trade without (or insufficient) reputation for relatively low amounts up to {0}, \
-  the likelihood of finding a trading partner is considerably reduced. This is because buyers tend to avoid \
-  engaging with sellers who lack reputation due to security risks.
-bisqEasy.tradeWizard.directionAndMarket.feedback.tradeWithoutReputation=Continue without reputation
+bisqEasy.tradeWizard.directionAndMarket.feedback.headline=You need reputation to sell!
```

Comment:
This sounds a bit weird to me (like reputation to/for sell).
Maybe better: "Build your reputation to start selling!" or "As seller, you need to have reputation"